BUILD_macos.md

```diff
@@ -77,9 +77,13 @@ PRs with corrections and updates are welcome!
   brew install pkg-config
   ```
 
-* To install `yarn` for building the UI
+* To install tools for building the UI:
   * `brew install node yarn`
   * Currently, [`yarn`](https://classic.yarnpkg.com/en/docs/install) (< 2.0.0) is required
+  * The `Rust` and `Cargo` version in [build.assets/Makefile](https://github.com/gravitational/teleport/blob/master/build.assets/versions.mk#L11) (search for `RUST_VERSION`) are required.
+  * The [`wasm-pack`](https://github.com/rustwasm/wasm-pack) version in [build.assets/Makefile](https://github.com/gravitational/teleport/blob/master/build.assets/versions.mk#L12) (search for `WASM_PACK_VERSION`) is required:
+    `curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh`
+  * [`binaryen`](https://github.com/WebAssembly/binaryen) is required: `brew install binaryen`. On debain-based linux, run `apt-get install binaryen`.
```

Comment:
I do not have binaryen and have not had a problem.. how do we explain that?

Reply:
I'm not sure exactly. I also don't recall having installed it before. It seems that we only need to install it explicitly when running linux ARM builds: 

https://github.com/gravitational/teleport/blob/7498827548221c844525b48a7544d804cb039c9d/build.assets/Dockerfile#L180

Reply:
What if we said that binaryen is needed only on arm64 Linux then and leave it at that?

Reply:
I think technically it's still needed (since we're always using wasm-opt), its just that it's somehow automatically included as a dependency on amd64 linux but not on arm64 (that's my educated guess, could be wrong).

How about this: 53924607649a2b04b13b798efadab038ad0d96d7

Reply:
`which wasm-opt` returns "wasm-opt not found" for me, but `yarn build-wasm` works with no problems. 😶

Reply:
I understand it better now: `wasm-pack build` installs a prebuilt `wasm-opt` automatically for linux x86_64, macos x86_64 && aarch64, and windows x86_64.

https://github.com/rustwasm/wasm-pack/blob/bdc4884fc5dcb50d78e521acfb2f613fb6588b48/src/install/mod.rs#L175-L183

Otherwise it fails.

ffda994a52051be61ebc522b5ede6f98cd605f83